backend/models/comments.js

```diff
@@ -16,6 +16,11 @@ const fetchCommentsByTourName = async (tour_Name) => {
     return results;
 }
 
+const fetchSpecificComment = async (username, tour_Name, content) => {
+    const results = await knex(COMMENT_TABLE).select().where({ content, username, username });
```

Comment:
2 usernames?

Reply:
oops my bad, that 2nd username should be tour_Name